models/marts/core/dim_customers.sql

```diff
@@ -55,14 +42,12 @@ with customers as (
 
         coalesce(customer_orders.number_of_orders, 0) as number_of_orders,
 
-        customer_and_payments.lifetime_value
+        customer_orders.lifetime_value
```

Comment:
You could consider a `coalesce(..., 0)` for `lifetime_value` as well. Do you think it makes more sense for customers with zero orders to have `null` or `$0` as their LTV? Either way is valid, I'd just err on the side of consistency with `number_of_orders`.